src/test/java/org/apache/commons/jxpath/ri/compiler/ExtensionFunctionTest.java

```diff
@@ -60,9 +59,9 @@ public void setUp() {
             lib.addFunctions(new ClassFunctions(TestFunctions2.class, "test"));
             lib.addFunctions(new PackageFunctions("", "call"));
             lib.addFunctions(
-                new PackageFunctions(
-                    "org.apache.commons.jxpath.ri.compiler.",
-                    "jxpathtest"));
+                    new PackageFunctions(
```

Comment:
What's changed here? There are too many formatting changes to make this simple to review.
In general, don't reformat existing code in PRs, it just makes noise in the PR.

Reply:
This reason is my IDE auto-reformating the file. I've only kept the relevant changes that are part of this PR.